modules_core/suggest-mentions.js

```diff
@@ -24,14 +25,18 @@ exports.create = function (api) {
     return function (cb) {
       if(!/^[%&@]\w/.test(word)) return cb()
```

Comment:
hmmm, do we really want the to be showing images for `%` projects ? 
also `&` blobs lookup don't seem to be working